src/main/java/org/jenkins/plugins/lockableresources/LockableResourcesManager.java

```diff
@@ -585,7 +585,7 @@ public synchronized void unreserve(List<LockableResource> resources) {
 			}
 
 			// continue with next context
-			LockStepExecution.proceed(resourceNamesToLock, nextContext.getContext(), nextContext.getResourceDescription(), null, false);
+			LockStepExecution.proceed(resourceNamesToLock, nextContext.getContext(), nextContext.getResourceDescription(), nextContext.getResources().requiredVar, false);
```

Comment:
may this be open to NPE?

Reply:
The exact same code is used on line 425 which has been in use for over a year.
See https://github.com/boblloyd/lockable-resources-plugin/commit/d0fb6a2862fc952802f3c2107acb813bf4ac22a1#diff-2cf9233da173bc9daaa43357bdb47d4f

Reply:
The underlying field is always initialized to non-null and never modified. So no NPE.

Reply:
in latest release in main branch in QueuedContextStruct its a List lockableResourcesStruct;
how can this be merged alone. LockStepExecution.proceed(resourceNamesToLock, nextContext.getContext(), nextContext.getResourceDescription(), nextContext.getResources().requiredVar, false);